### ISSUE TITLE
Error: Unable to resolve module ./App from node_modules/react-native-barcoder/index.js

### ISSUE THREAD

**Issue by aardvarkk** [TheBlindHawk/react-native-barcoder#6]
It seems to me that the latest package.json is broken. I'm unable to import the package in my React Native project with the Metro failure:

`Error: Unable to resolve module ./App from node_modules/react-native-barcoder/index.js`

I believe the cause is that the `main` entry points to `index.js`, which seems to contain the bones of an Expo project:

```
import { registerRootComponent } from 'expo'
import App from './App'

export default registerRootComponent(App)
```

I think two things need to be fixed:

- The index.js file at the root should not be included in the NPM package
- The package.json file should point to `dist/index.js` instead of just `index.js`

The workaround is to revert to v0.2.0 that does not have this issue.

**Comment by leoalho** [comment]
Hi, I'm facing the same issue, V0.2.0 works without problem. I skimmed through the source code quickly and I suspect the same cause that @aardvarkk mentioned.

**Comment by TheBlindHawk** [comment]
Hello thanks for opening the issue, and sorry for the late reply!

I must have accidentally added expo when testing locally, I'll remove it for the next version

**Comment by TheBlindHawk** [comment]
This issue should be fixed with v/0.2.2

I'll try to find a fix that doesn't require me to change the "main" when testing the code locally for future versions!
